src/control/PosController.cpp

```diff
@@ -80,8 +71,8 @@ void PosController::setAutoListenToInterface(bool listenToInterface) {
 
 void PosController::updatePid(pidVals pid) {
     if (lastPid != pid) {
-        xpid.setPID(pid);
-        ypid.setPID(pid);
+        this->pid = PidTwoAxesController(std::tuple_cat(pid,std::make_tuple(0)));
```

Comment:
This probably needs a comment that explains what the 0 and std::tuple_cat are doing. (Something short like //we always set F to 0)

Reply:
You can maybe also check for the existence of a fourth variable in the tuple in the constructor and put it to 0 there.

Reply:
That would be even prettier.